db/system_keyspace.cc

```diff
@@ -2662,6 +2663,31 @@ future<std::optional<mutation>> system_keyspace::get_group0_schema_version() {
     co_return std::nullopt;
 }
 
+future<std::optional<mutation>> system_keyspace::get_group0_schema_version() {
+    return get_scylla_local_mutation(_db, "group0_schema_version");
+}
+
+static constexpr auto SERVICE_LEVELS_STATUS_KEY = "service_level_v2_status";
+
+future<std::optional<mutation>> system_keyspace::get_service_levels_migration_status_mutation() {
+    return get_scylla_local_mutation(_db, SERVICE_LEVELS_STATUS_KEY);
+}
+
+mutation system_keyspace::make_service_levels_migration_status_mutation(const service::group0_guard& guard) {
+    auto s = _db.find_schema(db::system_keyspace::NAME, db::system_keyspace::SCYLLA_LOCAL);
+    auto* cdef = s->get_column_definition("value");
+    assert(cdef);
```

Comment:
Please change the assert to `on_internal_error`. I don't think the condition should ever fail here, but even if it fails it shouldn't be a critical failure warranting immediate crash of the node (`on_internal_error` throws an exception instead of crashing when compiled in release mode).

Reply:
with `query_processor::get_mutations_internal`  the assert doesn't exist